src/App.tsx

```diff
@@ -1,61 +1,129 @@
 import './App.scss';
+import { FormEvent, useState } from 'react';
+import { TodoList } from './components/TodoList/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [selectedTitle, setTitle] = useState('');
+  const [showTitleError, setTitleError] = useState(false);
+  const [showUserError, setUserError] = useState(false);
+  const [selectedUser, setUser] = useState('-1');
```

Comment:
it is recommended to use same names for value and setValue.
```suggestion
  const [selectedTitle, setSelectedTitle] = useState('');
  const [showTitleError, setTitleError] = useState(false);
  const [showUserError, setUserError] = useState(false);
  const [selectedUser, setSelectedUser] = useState('-1');
```